Bio/SeqIO/PdbIO.py

```diff
@@ -134,6 +134,10 @@ def PdbSeqresIterator(handle):
     # Late-binding import to avoid circular dependency on SeqIO in Bio.SeqUtils
     from Bio.SeqUtils import seq1
 
+    # raise exception if file is empty
+    if not handle.readline():
```

Comment:
This will read a line an discard it - so I would expect it to break the parser

Reply:
Can we use UndoHandle(handle).peekline() here instead?

Reply:
As a last resort - the fact this parser is a big for loop makes other approaches more complicated.

Reply:
One other way to do it is by making use of the fact that if the handle is empty, the loop `for line in handle:` would not be entered into. We can have something of the sort:
```
flag = False
for line in handle:
    flag = True
    ...

if not flag:
    raise ValueError("Empty file.")
```

I concede it's not the most elegant way to do it, but it's simple and standalone.

Reply:
That'd work. Maybe start with ``empty = True`` and inside the loop set ``empty = False``, but I like this approach more.

Reply:
Great! Thank you.